fireant/tests/dashboards/test_dashboard_api.py

```diff
@@ -63,8 +64,8 @@ def assert_slicer_queried(self, metrics, dimensions=None, mfilters=None, dfilter
             operations=operations or [],
         )
 
-    def assert_result_transformed(self, widgets, dimensions, mock_transformer, tx_generator, references=[],
-                                  operations=[]):
+    def assert_result_transformed(self, widgets, dimensions, mock_transformer, tx_generator, references=list(),
+                                  operations=list()):
```

Comment:
yes normally we would use lists but these are default values where the assertions require them to be list and not tuples.  Perhaps we could use a tuple for the default and cast them to a list.